test/deploy/rackhd_redfish_service_validator.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# This is a helper script to run the Redfish Service Validator on RackHD
+# Usage:
+# ./rackhd_redfish_service_validator.sh host:port
+
+if [ -z $1 ]; then
+  echo "Usage: ./rackhd_redfish_service_validator.sh host:port"
+  exit
+fi
+
+# set up virtual environment
+rm -rf rsv1
```

Comment:
Can you add some print statements/echo here - echo "Setting up virtual environment"